pkg/requests/util/util.go

```diff
@@ -3,13 +3,23 @@ package util
 import (
 	"net/http"
 
+	"github.com/google/uuid"
 	middlewareapi "github.com/oauth2-proxy/oauth2-proxy/v7/pkg/apis/middleware"
 )
 
+const (
+	XForwardedProto = "X-Forwarded-Proto"
+	XForwardedHost  = "X-Forwarded-Host"
+	XForwardedURI   = "X-Forwarded-Uri"
+	XRequestID      = "X-Request-Id"
```

Comment:
Oh I see why this is here now, I'm a bit conflicted. Do we import the other consts into other packages?

Reply:
I'm trying to remember why in this package I had `util_test` as a separate package when I made this suite for the other header helpers 🤔 

I think that's the main reason I made these public, let me play around with getting the test suite to share a package and get these private.

Reply:
Maybe my thinking was since this is `request utils` these are useful constants other packages could use to match up with out getters? I dunno, no I'm rationalizing for a past version of myself I don't remember 🤣

Reply:
Oh - I left a comment for myself why:

```
// TestRequestUtilSuite and related tests are in a *_test package
// to prevent circular imports with the `logger` package which uses
// this functionality
func TestRequestUtilSuite(t *testing.T) {
```

Reply:
Leave it public? Or make private and duplicate in the `util_test` package?

Reply:
Happy to leave it public, will likely come in useful somewhere else anyways